web/src/main/java/com/synopsys/integration/alert/web/api/job/JobConfigActions.java

```diff
@@ -230,18 +231,18 @@ public JobConfigActions(AuthorizationManager authorizationManager, DescriptorAcc
         }
     }
 
-    private ValidationResponseModel validateJobNameUnique(@Nullable UUID currentJobId, JobFieldModel jobFieldModel) {
-        ValidationResponseModel responseModel = ValidationResponseModel.success("Valid name");
+    private List<AlertFieldStatus> validateJobNameUnique(@Nullable UUID currentJobId, JobFieldModel jobFieldModel) {
+        List<AlertFieldStatus> fieldStatuses = new ArrayList<>();
```

Comment:
There should only be 1 name per job, so why are we returning a list for this method?

Reply:
I think your right, I saw the the for loop and incorrectly thought it should have been returned as a list because it looped through all of the fieldModels. I can change this to return a single AlertFieldStatus.